sklearn/gaussian_process/kernels.py

```diff
@@ -44,6 +44,17 @@ def _check_length_scale(X, length_scale):
     return length_scale
 
 
+def _check_offset(X, c):
```

Comment:
This seems to be identical to `_check_length_scale `? Essentially it is checking whether the kernel is anisotropic by checking `length_scale` or `c` in your case.